src/asli/asli.py

```diff
@@ -57,9 +58,7 @@ def get_lows(da: xr.DataArray, mask: xr.DataArray) -> pd.DataFrame:
     sector_mean_pres = asl_sector_mean(da, mask)
     threshold = sector_mean_pres
 
-    date = datetime.datetime.strptime(str(da.time.values), "%Y%m%d")
-
-    time_str = date.strftime("%Y-%m-%d")
+    time_str = np.datetime_as_string(da.valid_time, unit='D')
```

Comment:
I'd prefer not to import numpy here for this sole purpose, unless there's a reason that's not obvious to me.
If the date formats are `2024-01-01T00:00:00.000000000` we could maybe use `datetime.strptime(str(da.valid_time), "%Y-%m-%dT%H:%M:%S.%f000").date()` or something that actually ignores that empty time string.

Reply:
I did that because `da.valid_time.values` is a numpy datetime64 dtype:

```python
array(['2024-01-01T00:00:00.000000000', '2024-02-01T00:00:00.000000000', '2024-03-01T00:00:00.000000000',
       '2024-04-01T00:00:00.000000000', '2024-05-01T00:00:00.000000000', '2024-06-01T00:00:00.000000000',
       '2024-07-01T00:00:00.000000000', '2024-08-01T00:00:00.000000000', '2024-09-01T00:00:00.000000000'],
      dtype='datetime64[ns]')
```

I wasn't sure if that was new, but from a bit of googling strptime can't deal with those, so I'm guessing it is. I can convert it to a datetime with pandas instead (already imported) of numpy though

Reply:
Ah I see, I'm still more familiar with the stdlib over numpy/pandas world so this makes sense given how they'll be processed downstream anyway so fine by me, thanks!

Reply:
Changed:

```python
time_str = np.datetime_as_string(da.valid_time, unit='D')
```

to

```python
datetime_values = pd.to_datetime(da.valid_time.values)
time_str = datetime_values.strftime("%Y-%m-%d")
```